tests/sentry/api/endpoints/test_organization_metrics.py

```diff
@@ -183,6 +161,78 @@ def test_metrics_index(self):
             {"name": "session.init", "type": "numeric", "operations": [], "unit": "sessions"},
         ]
 
+    @with_feature(FEATURE_FLAG)
+    def test_metrics_index(self):
+        """
+
+        Note that this test will fail once we have a metrics meta store,
+        because the setUp bypasses it.
+        """
+        response = self.get_success_response(self.organization.slug, project=[self.project.id])
+
+        assert response.data == [
+            {"name": "metric1", "type": "counter", "operations": ["sum"], "unit": None},
+            {"name": "metric2", "type": "set", "operations": ["count_unique"], "unit": None},
+            {"name": "metric3", "type": "set", "operations": ["count_unique"], "unit": None},
+        ]
+
+        self.store_session(
+            self.build_session(
+                project_id=self.proj2.id,
+                started=(time.time() // 60) * 60,
+                status="ok",
+                release="foobar@1.0",
+            )
+        )
+
+        response = self.get_success_response(self.organization.slug, project=[self.proj2.id])
+        assert response.data == self.session_metrics_meta
+
+    @with_feature(FEATURE_FLAG)
+    def test_metrics_index_invalid_derived_metric(self):
+        for errors, minute in [(0, 0), (2, 1)]:
+            self.store_session(
+                self.build_session(
+                    project_id=self.proj2.id,
+                    started=(time.time() // 60 - minute) * 60,
+                    status="ok",
+                    release="foobar@2.0",
+                    errors=errors,
+                )
+            )
+
+        DERIVED_METRICS.update(
```

Comment:
I'd use `mock.patch` or pytest's monkeypatch here. this has the potential to influence other tests within the same testrun, potentially in entirely different files.

Reply:
Yeah good point.. I will fix those in the upcoming PR